models/parser_jobs_tut_by.py

```diff
@@ -0,0 +1,197 @@
+              'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=python&page=0'
+        response = HTTPClient.get(url, headers=headers).text
+        lxml_text = RabotaByParser.get_lxml(response)
+        list_id = RabotaByParser.get_list_of_vacancy_id(lxml_text)
+        first_vacancy_id = list_id[0]
+        response_for_first_vacancy = HTTPClient.get(f'https://rabota.by/vacancy/{first_vacancy_id}?query=python',
+                                                    headers=headers).text
+        description = RabotaByParser.find_vacancy_description(response_for_first_vacancy)
+        return description.text.lower()
+
+    @staticmethod
+    def get_avg_occurrences_for_words():
+        """
+        Returns average amount of occurrences for words python, linux, flask
+        """
+        headers = {
+            'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/webkit-version (KHTML, like Gecko) '
+                          'Silk / browser - version like Chrome / chrome - version Safari / webkit - version'}
+        url_for_page = 'https://rabota.by/search/' \
+                       'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=python&page='
+        url = 'https://rabota.by/search/' \
+              'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=python&page=0'
+        response = HTTPClient.get(url, headers=headers).text
+        pages = RabotaByParser.amount_of_pages(response)
+        page = 0
+        all_urls_list = []
+        while page <= int(pages) - 1:
+            response = HTTPClient.get(f'{url_for_page}{page}', headers=headers).text
+            lxml_text = RabotaByParser.get_lxml(response)
+            list_id = RabotaByParser.get_list_of_vacancy_id(lxml_text)
+            list_urls_for_one_page = RabotaByParser.get_list_of_url_of_vacancies_pages_by_id(list_id)
+            all_urls_list += list_urls_for_one_page
+            page += 1
+        dictionary = {}
+        number = 1
+        python = 0
+        linux = 0
+        flask = 0
+        for ur in all_urls_list:
+            response = HTTPClient.get(ur, headers=headers).text
+            description = RabotaByParser.find_vacancy_description(response)
+            description = description.text.lower()
+            count_python = RabotaByParser.count_word('python', description)
+            count_linux = RabotaByParser.count_word('linux', description)
+            count_flask = RabotaByParser.count_word('flask', description)
+            dictionary[f'{number} vacancy'] = {'python': count_python, 'linux': count_linux, 'flask': count_flask}
+            python += count_python
+            linux += count_linux
+            flask += count_flask
+            number += 1
+        avg_python = python // len(all_urls_list)
+        avg_linux = linux // len(all_urls_list)
+        avg_flask = flask // len(all_urls_list)
+        return avg_python, avg_linux, avg_flask
```

Comment:
It is awful method!!! Create a method which receives a word as argument and counts it. ONLY ONE WORD!